data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -318,6 +318,52 @@ then
     record_service_stage_success
 fi
 
+if [ ! -f cmdbootline-nto-bootstrap.done ]; then
+	record_service_stage_start "cmdbootline-nto-bootstrap"
+
+	rm --recursive --force cmdbootline-nto-bootstrap
```

Comment:
```suggestion
	rm --recursive --force cmdbootline-nto-bootstrap
	mkdir cmdbootline-nto-bootstrap
```
I don't see anywhere else we are creating this directory, but we need it as our --asset-output-dir

Reply:
No need to do this. NTO would make sure output dir exists or it will create it.
https://github.com/openshift/cluster-node-tuning-operator/blob/da99c2c5da41c021c6396351a29714bc1003dd90/pkg/tuned/cmd/render/render.go#L73-L77